docs/variants/novacustom_ns5x_7x/releases.md

```diff
@@ -54,6 +54,15 @@ Test results for this platform can be found
 
 See how to verify signatures on [this video](https://asciinema.org/a/518379)
 
+### EC firmware transition
+
+Note that, version 1.3.0 of Dasharo firmware works correctly only with the
+open EC firmware.
+
+Transition to the open EC firmware might be done:
+
+- by using [EC flashing kit](ec_recovery.md).
```

Comment:
@sulewskiprzemyslaw of course we should add some disclaimer that flashing new firmware without EC may lead to bricking system and we are not responsible for such demage.

Reply:
added